pkgs/by-name/ff/fflogs/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, appimageTools
+, fetchurl
+}:
+
+let
+  pname = "fflogs";
+  version = "8.3.42";
+  src = fetchurl {
+    url = "https://github.com/RPGLogs/Uploaders-fflogs/releases/download/v${version}/fflogs-v${version}.AppImage";
+    sha256 = "08mkixjpppijssmg63l542xhmxyr7xs9z1v9qfjda7cfqgah5kh5";
```

please use the SRI hash format

```suggestion
    hash = "sha256-...";
```